### PR TITLE
add nzaom app

### PR DESCRIPTION
**Pull request by daemonchen**

### CODE REVIEW

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -501,6 +501,13 @@ var apps = [
     linkAppStore: 'https://itunes.apple.com/us/app/tian-cai-duan-zi-shou-shen/id992312701?l=zh&ls=1&mt=8',
     linkPlayStore: 'https://play.google.com/store/apps/details?id=com.geniusJokeWriter.app',
     author:'Ran Zhao&Ji Zhao'
+  },
+  {
+    name:'你造么',
+    icon:'http://7xk1ez.com2.z0.glb.qiniucdn.com/logo-mobile-0114logo_welcom.png',
+    linkAppStore: 'https://itunes.apple.com/us/app/ni-zao-me/id1025294933?l=zh&ls=1&mt=8',
+    linkPlayStore: '',
```
Comment (ide) [inline website/src/react-native/showcase.js]: can you remove this and use just the `link` key?

**Comment by satya164** [comment]
Can you squash your commits to one?

File: website/src/react-native/showcase.js [hunk 2]
```diff
@@ -507,6 +507,12 @@ var apps = [
     linkAppStore: 'https://itunes.apple.com/us/app/tian-cai-duan-zi-shou-shen/id992312701?l=zh&ls=1&mt=8',
     linkPlayStore: 'https://play.google.com/store/apps/details?id=com.geniusJokeWriter.app',
     author:'Ran Zhao&Ji Zhao'
+  },
+  {
+    name:'你造么',
+    icon:'http://7xk1ez.com2.z0.glb.qiniucdn.com/logo-mobile-0114logo_welcom.png',
```
Comment (brentvatne) [inline website/src/react-native/showcase.js]: space between `:` and `'` please

**Comment by daemonchen** [comment]
@brentvatne I just add space between `:` and `'`, but the CI was failed. Is there any wrong I made?

**Comment by corbt** [comment]
@daemonchen no the failure isn't related to your changes. I'll merge this now, congrats on publishing an RN app!

**Comment by mosch** [comment]
@facebook-github-bot this is so annoying.

**Pull request merged**